broker/consumer/ligo/consumer.properties

```diff
@@ -0,0 +1,11 @@
+# see https://docs.confluent.io/platform/current/kafka/authentication_sasl/authentication_sasl_oauth.html#configuring-ak-clients
+
+bootstrap.servers=kafka.gcn.nasa.gov:9092
+security.protocol=SASL_SSL
+sasl.mechanism=OAUTHBEARER
+sasl.oauthbearer.token.endpoint.url=https://auth.gcn.nasa.gov/oauth2/token
+sasl.jaas.config=org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required \
+     unsecuredLoginStringClaim_sub="pitt-google-broker-test" \
+     oauth.client.id="CLIENT_ID" \
+     oauth.client.secret="CLIENT_SECRET" \
+     scope="gcn.nasa.gov/kafka-public-consumer";
```

Comment:
This `OAuthBearerLoginModule` creates unsecured tokens -- it does not accept the client ID, secret, and scope options. (See your link above for the list of options that it does take.)

Once you get to the point where you *can* plug in those options, it will be important to see the code you're using to substitute values for the `CLIENT_ID` and `CLIENT_SECRET`. That is an operation where things can easily go wrong.